lib/daffy_lib/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module DaffyLib
-  VERSION = "0.1.9"
+  VERSION = "0.1.10"
```

Comment:
Vulnerability updates for rails-sanitizer and nokogiri were merged since @michellefoy's last version bump, so we should bump again.